torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -3874,4 +3920,4 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::_allgather_base(
 
 } // namespace c10d
 
-#endif // USE_C10D_NCCL
```

Comment:
nit: maybe not expected?

Reply:
lintrunner ate this I think, hmm.  fixing.